### ISSUE TITLE
Fix Footer Typo

### ISSUE THREAD

**Issue by marisabrantley** [youtanimstar/music-hub#31]
This great website has a typo in its footer.

It has the word "CopyRights" instead of "Copyright".

Please assign this to me under the Hacktoberfest label so this site can look even more polished.

**Comment by marisabrantley** [comment]
<img width="474" alt="Screen Shot 2023-10-17 at 9 03 27 PM" src="https://github.com/youtanimstar/music-hub/assets/60168324/aaf5aa06-dbf2-4ed0-aea6-f3860e939ee5">

**Comment by youtanimstar** [comment]
> This great website has a typo in its footer.
> 
> It has the word "CopyRights" instead of "Copyright".
> 
> Please assign this to me under the Hacktoberfest label so this site can look even more polished.

Indeed, these minor changes may not significantly enhance the project. If you wish to proceed, you may do so. @marisabrantley
